patches/runtime/0033-Build-net4-in-source-build.patch

```diff
@@ -0,0 +1,83 @@
+  /work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.Configuration.ConfigurationManager/ref/System.Configuration.ConfigurationManager.cs(1471,125):
+    error CS0433: The type 'PermissionSet' exists in both 'System.Security.Permissions, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.Configuration.ConfigurationManager/ref/System.Configuration.ConfigurationManager.csproj]
+  /work/.dotnet/sdk/5.0.100-rc.1.20452.10/Microsoft.Common.CrossTargeting.targets(88,5): error MSB4181: The "MSBuild" task returned false but did not log an error. [/work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.Configuration.ConfigurationManager/ref/System.Configuration.ConfigurationManager.csproj]
+  /work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.ServiceProcess.ServiceController/ref/System.ServiceProcess.ServiceController.cs(39,43):
+    error CS0433: The type 'EventLog' exists in both 'System.Diagnostics.EventLog, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.ServiceProcess.ServiceController/ref/System.ServiceProcess.ServiceController.csproj]
+  /work/.dotnet/sdk/5.0.100-rc.1.20452.10/Microsoft.Common.CrossTargeting.targets(88,5): error MSB4181: The "MSBuild" task returned false but did not log an error. [/work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.ServiceProcess.ServiceController/ref/System.ServiceProcess.ServiceController.csproj]
+---
+ src/libraries/Directory.Build.props                          | 5 +++--
+ .../src/Microsoft.Extensions.Configuration.Xml.csproj        | 3 +++
+ .../ref/System.Configuration.ConfigurationManager.csproj     | 2 +-
+ .../ref/System.ServiceProcess.ServiceController.csproj       | 2 +-
+ 4 files changed, 8 insertions(+), 4 deletions(-)
+
+diff --git a/src/libraries/Directory.Build.props b/src/libraries/Directory.Build.props
+index 7a3946c28a4..dfe161d6d72 100644
+--- a/src/libraries/Directory.Build.props
++++ b/src/libraries/Directory.Build.props
+@@ -41,8 +41,9 @@
+     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+     <BuildTargetFramework Condition="'$(BuildTargetFramework)' == '' and '$(TargetFramework)' != ''">$(TargetFramework)</BuildTargetFramework>
+     <!-- Build all .NET Framework configurations when net48 is passed in. This is for convenience. -->
```

Comment:
What I'm missing is why is net48 special?  Are we passing this in somewhere else?  Or is it the TFM that is used in the projects that we want to build below?  What happens if it changes and these get missed?

Reply:
It's not special for us, it's only special in dotnet/runtime's current source code per this comment. I just made the patch reuse dotnet/runtime's list rather than having our own copy of the list that just adds in `net48`. If it gets out of date (`net49`? 😮), the patch will conflict, and we will be forced to update it. I'll add some info to the patch description with another PR after merging this one.